build.gradle

```diff
@@ -45,6 +45,7 @@ ext {
     buildToolsVersion = "29.0.2"
     compileSdkVersion = 29
     minSdkVersion = 16
+    nav_version = "2.3.0"
```

Comment:
For no particular reason, none of the variables in this file have underscores, but to be consistent, I'd just call this `navigationversion`.  Also, it's probably best to move it right below `awsSdkVersion`, so that it's with the other dependencies.  These values here at the top (`minSdkVersion`, `targetSdkVersion`, etc), aren't related to dependencies.